src/bundle/Resources/public/js/scripts/admin.location.add.custom_url.js

```diff
@@ -22,8 +23,19 @@
             input.value = '';
             toggleButtonState();
         };
+        const toggleSiteAccessSelect = (event) => {
+            const checkbox = event.target;
+            const isChecked = checkbox.checked;
+
+            const siteAccessSelect = modal.querySelector('[name="custom_url_add[site_access]"]');
+
+            isChecked
+                ? siteAccessSelect.removeAttribute('disabled')
+                : siteAccessSelect.setAttribute('disabled', true)
+        };
```

Comment:
```suggestion
        const toggleSiteAccessSelect = (event) => {
            const isChecked = event.target.checked;
            const siteAccessSelect = modal.querySelector('[name="custom_url_add[site_access]"]');
            const methodName = isChecked ? 'removeAttribute' : 'setAttribute';
            
            siteAccessSelect[methodName]('disabled', true);
        };
```

This would be more in line with our standards.